tutorials/tutorial1_sinus-interactive.ipynb

```diff
@@ -4,10 +4,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# DECOMON tutorial #1\n",
-    "\n",
+    " # <span style='background:purple'> <span style='color:White'> DECOMON tutorial #1 </span></span>\n",
+    " \n",
     "_**Bounding the output of a Neural Network trained on a sinusoidal function**_\n",
     "\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"data/decomon.png\" alt=\"Decomon!\" width=\"100\">\n",
+    "</p>\n",
+    "\n",
+    "\n",
+    "\n",
+    "- &#x1F4DA; <a href=\"https://airbus.github.io/decomon\"> <span style='color:purple'>Documentation</span> </a>\n",
+    "- <a href=\"https://github.com/airbus/decomon\"> <span style='color:purple'>Github</span> </a>\n",
+    "- <a href=\"https://github.com/airbus/decomon/blob/master/TUTORIALS.md\"> <span style='color:purple'>More **decomon** tutorials</span> </a>\n",
```

Comment:
That link does not work at all:
- no branch "master"
- no TUTORIALS.md file

Maybe we could point to https://airbus.github.io/decomon/main/tutorials.html ?